calibimg.rst

```diff
@@ -55,26 +55,26 @@ Parameters
 	
 
 
-Subtasks
---------
+Subtask Targets
+----------------
 
-- 	``refObjLoader`` - default target=LoadAstrometryNetObjectsTask -   reference object loader
+- 	``refObjLoader`` - default=LoadAstrometryNetObjectsTask -   reference object loader
```

Comment:
Removed the extra and now unneeded word 'target' from the defaults for all of these subtask target descriptions.